### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by raihan2bd**
To make my project look good I use (responsive design) technology and add some @media queries, for that I make changes in my project HTML, and CSS, and also add some images. 👍

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -181,131 +192,142 @@ <h2 class="work_title">Multi-Post Stories</h2>
...
+            <h3 class="sub_title">
+              Skills
+              <img
+                class="arrow_icon"
+                src="./images/right-arrow-icon.png"
+                alt="down"
+              />
+            </h3>
+          </ul>
         </div>
```
Comment (Qoosim) [inline index.html:287]: - [ ] It is required that on the desktop version of your portfolio, the list of languages should be on a row and each language icon and text should be on a column as we have it on the template you chose. Kindly compare the two designs and work your design in line with that of the template you chose.

### Your Design

your-design-lang


### Figma Design

figma-design-lang


- [ ] You're expected to produce a close replica of the template you chose. This involves font, color, margin, padding among others. Kindly look into your headline section of your page on the desktop version and work on the margin of the section. Take a look at both designs and work your design along side the template of your choice.

### Your Design

your-design-headline


### Figma Design

figma-design-headline

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @raihan2bd, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-   No linter errors. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 
-   Good README file. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -23,62 +23,78 @@
...
+                    alt="Github"
+                /></a>
+              </li>
+              <li class="social_link">
+                <a href="#"
+                  ><img class="social_icon" src="./images/doit.svg" alt="Diot"
+                /></a>
+              </li>
+            </ul>
+          </div>
```
Comment (ichala) [inline index.html:92]: - [ ] as requested by the previous reviewer kindly try to follow the Figma design for the headline section, adding some paddings and margins will help you match the Figma design 

**your design**
image

**Figma Design**

image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @raihan2bd 
Good job so far! 
👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Great Design✔️ 


 ## Required Changes ♻️

- [ ] 🛑 **_Check the comments below_**

## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**